salt/kubelet/kubelet.jinja

```diff
@@ -32,5 +44,6 @@ KUBELET_ARGS="\
     --cloud-provider=aws \
 {% endif -%}
     {{ pillar['components']['kubelet']['args'] }} \
+    --node-labels={{ node_labels|join(',') }} \
```

Comment:
(The ansible we used is here as a reference if needed https://gitlab.suse.de/caasp-ha/hcp-ansible-poc/tree/master/ansible/roles/kubelet )